nebula/ui/components/MZStepProgressBar.qml

```diff
@@ -70,10 +72,22 @@ Item {
                     }
                 }
 
-                onClicked: activeIndex = index
+                onClicked: {
+                    activeIndex = index
+                    buttonClicked(index)
+                }
 
-                accessibleName: currentState === MZStepProgressBarDelegate.State.Complete ? MZI18n.OnboardingProgressBarAccessibilityStepComplete.arg(labelText).arg(index + 1).arg(progressBar.model.count)
-                                                                                          : MZI18n.OnboardingProgressBarAccessibilityStepCurrent.arg(labelText).arg(index + 1).arg(progressBar.model.count)
+                accessibleName: {
```

Comment:
On Windows when I use the screen reader next command (CapsLock+RightArrow in Narrator), it doesn't go through the incomplete steps. Since a sighted user can see the incomplete steps, a screen reader user should also be able to access it using the next command. (The next command is different from TAB, which moves to the next focusable item.) This can be addressed in a future PR if needed.